config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml

```diff
@@ -233,6 +233,45 @@ presubmits:
 
     annotations:
       testgrid-create-test-group: 'true'
+  - name: pull-kubernetes-e2e-gce-serial
+    always_run: false
+    optional: true
+    branches:
+    - master
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+      preset-bazel-scratch-dir: "true"
+      preset-pull-kubernetes-e2e: "true"
+      preset-pull-kubernetes-e2e-gce: "true"
+    spec:
+      containers:
+      - args:
+        - --root=/go/src
+        - --repo=k8s.io/kubernetes=$(PULL_REFS)
+        - --repo=k8s.io/release
+        - --upload=gs://kubernetes-jenkins/pr-logs
+        - --timeout=150
+        - --scenario=kubernetes_e2e
+        - --
+        - --build=bazel
+        - --cluster=
+        - --extract=local
+        - --gcp-node-image=ubuntu
```

Comment:
Is Ubuntu intentional? Most other jobs run cos.

Reply:
not intentional, I copied from `pull-kubernetes-e2e-gce-iscsi-serial` job.
I've updated it to use `gci`. I guess `gci` means `cos`?